comm/src/main/scala/coop/rchain/p2p/effects/TransportLayer.scala

```diff
@@ -0,0 +1,44 @@
+package coop.rchain.p2p.effects
+
+import scala.concurrent.duration.{Duration, MILLISECONDS}
+
+import cats.Monad
+import cats.data.EitherT
+
+import coop.rchain.catscontrib.Catscontrib._
+import coop.rchain.catscontrib.MonadTrans
+import coop.rchain.comm.CommError.CommErr
+import coop.rchain.comm.{PeerNode, ProtocolMessage, ProtocolNode}
+
+trait TransportLayer[F[_]] {
+  def roundTrip(msg: ProtocolMessage,
+                remote: ProtocolNode,
+                timeout: Duration): F[CommErr[ProtocolMessage]]
+  def local: F[ProtocolNode]
+  def commSend(msg: ProtocolMessage, peer: PeerNode): F[CommErr[Unit]]
+  def broadcast(msg: ProtocolMessage): F[Seq[CommErr[Unit]]]
+}
+
+object TransportLayer extends TransportLayerInstances {
+  def apply[F[_]](implicit L: TransportLayer[F]): TransportLayer[F] = L
+
+  def forTrans[F[_]: Monad, T[_[_], _]: MonadTrans](
+      implicit C: TransportLayer[F]): TransportLayer[T[F, ?]] =
+    new TransportLayer[T[F, ?]] {
+      def roundTrip(
+          msg: ProtocolMessage,
+          remote: ProtocolNode,
+          timeout: Duration = Duration(500, MILLISECONDS)): T[F, CommErr[ProtocolMessage]] =
```

Comment:
This default value should not be here. Please remove because apparently I forgot

Reply:
ok